src/components/ErrorPage/ErrorPage.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+
+export default function ErrorPage() {
```

Comment:
Is error page solely for 404 error or does it encapsulate all kinds of errors?

Reply:
Right now it's kind of a catch all, we can tweak once we have better handling in the future

Reply:
Sounds good